test/test_files/abl_godunov_hypre/abl_godunov_hypre.i

```diff
@@ -66,19 +66,46 @@ zhi.type =   "slip_wall"
 zhi.temperature_type = "fixed_gradient"
 zhi.temperature = 0.003 # tracer is used to specify potential temperature gradient
 
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#              HYPRE                    #
+#.......................................#
+
+mac_proj.verbose = 1
+mac_proj.mg_verbose = 1
+mac_proj.mg_cg_verbose = 1
```

Comment:
```suggestion
mac_proj.verbose = 0
```

I recommend we set `verbose = 0` for regression tests, as it is also used as example files for users. `mg_verbose` was replaced with `verbose`.  We removed `mg_cg_verbose` in favor of `bottom_verbose`.